src/app/app.component.html

```diff
@@ -0,0 +1,71 @@
+<!--The content below is only a placeholder and can be replaced.-->
+
+<form #signup="ngForm" (ngSubmit)="seveinfo(signup)">
+  <div class="panel panel-primary">
+    <div class="panel-heading">
+      <h3 class="panel-title">Sign-up form</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group" [class.has-error]="email.invalid && email.touched">
+        <label for="email" class="control-label">Email</label>
+        <input id="email" type="text" class="form-control" required email name="email"
+        [formControl]="email">
+        <span class="help-block"
+        *ngIf="email.errors && email.errors.required && email.touched">
+        Email is required
+        </span>
+        <span class="help-block"
+        *ngIf="email.errors && email.errors.email && email.touched">
+        Email is invalid
+        </span>
+
+      </div>
+        <label for tips="ipasumatips" class="control-label">Īpašuma tips</label><br>
+        <mat-radio-group name="ipasumatips" [(ngModel)]="ipasumatips">
+          <mat-radio-button value="dzivokli" checked>Dzīvoklis</mat-radio-button><br>
+          <mat-radio-button value="māju">Māja</mat-radio-button><br>
+          <mat-radio-button value="zemi">Zeme</mat-radio-button><br>
+          <mat-radio-button value="mežu">Mežš</mat-radio-button><br>
```

Comment:
It's ok for labels (visual texts) to be in Latvian, but every input value used by the system should be in English.

```suggestion
          <mat-radio-button value="forest">Mežš</mat-radio-button><br>
```

Reply:
I agree, but I made so these values show up on next radio buttons when chosen. But there always a better way of doing that...

Reply:
The labels will still appear next to the radio buttons in Latvian. What this does (see example above) is set the values in English. These are invisible to people and only visible to machines :)